asymmetric/endpoints.py

```diff
@@ -26,7 +26,6 @@ def __init__(
         self.__route: str = route
         self.__method: str = method
         self.__function: Callable[..., Any] = function
-        self.__decorated_function: Callable[..., Any] = decorated_function
```

Comment:
This change should include the removal of the `decorated_function` from the methods that require it, so that there are no unused parameters